css/style.css

```diff
@@ -0,0 +1,897 @@
+  font-size: 70px;
+  line-height: 85px;
+  color: #E51937;
+  left: 80px;
+  top: 0;
+  margin: 30px 0;
+}
+
+.box {
+  width: 440px;
+  height: 900px;
+  top: 0;
+  right: 0;
+  background: #E51937;
+  color: #fff;
+  position: absolute;
+}
+
+.box-right__inner {
+  width: 320px;
+  margin: 0 auto;
+  margin-top: 50px;
+}
+
+.title {
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  margin-bottom: 12px;
+}
+
+.text {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  margin-bottom: 55px;
+}
+
+.about-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 267px;
+  margin-bottom: 171px;
+}
+
+.about__feature {
+  display: flex;
+  align-items: center;
+}
+
+.icon04 {
+  background: url(../img/icon04.svg) center no-repeat;
+  /* background-repeat: no-repeat;
+  background-position: center; */
+  margin-right: 24px;
+  width: 36px;
+  height: 53px;
+}
+
+.icon05 {
+  background: url(../img/icon05.svg) center/contain no-repeat;
+  /* background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain; */
+  margin-right: 20px;
+  width: 55px;
+  height: 34px;
+}
+
+.icon06 {
+  background: url(../img/icon06.svg) center no-repeat;
+  margin-right: 24px;
+  width: 36px;
+  height: 52px;
+}
+
+.order {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  left: 50px;
+  bottom: 60px;
+}
+
+.info-icon {
+  background: url(../img/info-icon.svg) center top no-repeat;
+  margin-right: 10px;
+  width: 22px;
+  height: 22px;
+}
+
+.info {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  margin-right: 28px;
+}
+
+.info:hover {
+  font-size: 14px;
+  margin-top: -3px;
+  margin-right: 18px;
+}
+
+.btn-white {
+  border-color: #fff;
+  color: #fff;
+  background: #E51937;
+}
+
+.btn-white:hover {
+  color: #E51937;
+  background: #fff;
+}
+
+/* media for section 3 */
+
+.review__wrapper {
+  height: 900px;
+  position: relative;
+}
+
+.review-img {
+  background: url(../img/models.svg) bottom right no-repeat;
+  height: 300px;
+  max-width: 100%;
+}
+
+.img3 {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  height: 900px;
+  margin-right: 46px;
+  z-index: 2;
+}
+
+.image3 {
+  max-width: 100%;
+}
+
+.box-left {
+  left: 0;
+}
+
+.box-left__inner {
+  width: 320px;
+  margin: 0 auto;
+  margin-top: 50px;
+}
+
+.text-left {
+  margin-bottom: 50px;
+}
+
+.review-score {
+  width: 300px;
+}
+
+.review-title {
+  margin-top: 49px;
+  margin-bottom: 26px;
+}
+
+.score, .review-security {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 90px;
+}
+
+.score__list {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 21px;
+  text-transform: uppercase;
+}
+
+li > span {
```

Comment:
Codacy found an issue: [Expected selector "li > span" to come before selector ".info-title > span" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/finalproject/pullRequest?prid=5923513)